domains/lens.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "imlens",
+        "email": "xlunarr@hotmail.com"
+    },
+    "record": {
+        "CNAME": "lens"
```

Comment:
Invalid CNAME record.

Reply:
what was the CNAME supposed to be again?

Reply:
A valid hostname without any URL schemes attached.

E.g: beta.dibster.com

Reply:
would imlens.github.io work?